ui/framework/src/ui-framework/childwindow/ChildWindowManager.tsx

```diff
@@ -163,13 +163,14 @@ export class ChildWindowManager {
     const outLocation = { ...location };
     if (0 === location.top && 0 === location.left) {
       center = center ?? true;
+      const windowTop = window.top ?? window;

```

Comment:
@DanEastBentley @bsteinbk TypeScript was complaining below (about `window.top`):

> Object is possibly 'null'.  **ts(2531)**

Is this safe?  Or should we do something else when `window.top` is null?

Reply:
That fix looks good to me.